framework/generated/dx12_generators/dx12_struct_decoders_to_json_body_generator.py

```diff
@@ -63,11 +63,29 @@ def beginFile(self, gen_opts):
             #include "graphics/dx12_util.h"
             #include "util/json_util.h"
 
+            /// @file Implementations of functions for converting decoded D3D12 structs to JSON.
+
             GFXRECON_BEGIN_NAMESPACE(gfxrecon)
             GFXRECON_BEGIN_NAMESPACE(decode)
 
             using util::JsonOptions;
 
+            /// @defgroup ManualD3D12StructFieldToJsons Manual functions to convert structs.
+            /** @{ */
+            static void FieldToJson(nlohmann::ordered_json& jdata, const D3D12_RENDER_PASS_BEGINNING_ACCESS_PRESERVE_LOCAL_PARAMETERS& data, const JsonOptions& options)
```

Comment:
No time to change this now, but I hate writing code in python that just appears verbatim in a generated file.  This should have a TODO to move to a non-generated file so there's less steps required for updating.

Reply:
That's what we do in other consumer bases.  No worries.  Maybe just a preference thing.  I hate having to go figure out where code is and I have banged my head on things like this in the past where code gen wasn't updating the function until I found out it was special cased inside the python script.  But I could also see the reverse scenario.